### PR TITLE
enh(release): Add Centreon Gorgone 21.10.1

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [ ] 21.04.x
- [x] 21.10.x
- [ ] 22.04.x (master)

### CODE REVIEW

File: en/releases/centreon-core.md [hunk 1]
```diff
@@ -121,6 +121,13 @@ By:
 
 ## Centreon Gorgone
 
+### 21.10.1
+
+#### Bugfixes
+
+- Make Gorgone’s private key readable by centreon-gorgone only
+- Fixed restart was too long and occasionally reaches the max systemctl timeout
```
Comment (mdebauchez) [inline en/releases/centreon-core.md:129]: The description used in Release Note field is better I think : 
Gorgone was too long to restart, which could cause the service to reach the systemctl timeout.

File: fr/releases/centreon-core.md [hunk 2]
```diff
@@ -120,6 +120,14 @@ By:
 
 ## Centreon Gorgone
 
+### 21.10.1
+
+#### Bugfixes
+
+- Make Gorgone’s private key readable by centreon-gorgone only
+- Fixed restart was too long and occasionally reaches the max systemctl timeout
```
Comment (mdebauchez) [inline fr/releases/centreon-core.md:128]: Same comment as in english version

**Review by mdebauchez — CHANGES_REQUESTED** [review]

File: en/releases/centreon-core.md [hunk 3]
```diff
@@ -121,6 +121,13 @@ By:
 
 ## Centreon Gorgone
 
+### 21.10.1
+
```
Comment (tuntoja) [inline en/releases/centreon-core.md:125]:
```suggestion

Release date: `December 14, 2021`

```

File: fr/releases/centreon-core.md [hunk 4]
```diff
@@ -120,6 +120,14 @@ By:
 
 ## Centreon Gorgone
 
+### 21.10.1
+
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:124]:
```suggestion

Release date: `December 14, 2021`

```

**Review by tuntoja — CHANGES_REQUESTED** [review]

**Review by tuntoja — APPROVED** [review]

**Pull request merged**